src/Components/DrillListPage.js

```diff
@@ -33,13 +37,14 @@ export const DrillListPage = props => {
           keyExtractor={item => item.id.toString()}
           renderItem={({ item }) => (
             <TouchableOpacity style={styles.drill} onPress={() => navigation.navigate('DrillPage', { drill: item })}>
-              <Image style={styles.image} source={{ uri: item.image }} />
+              <ImageBackground source={{ uri: item.image }} style={styles.image} imageStyle={styles.imageOpacity}>
+                <Text style={{ ...styles.imageText, ...styles.imageTextMain }}>{item[imageMainData]}+</Text>
```

Comment:
Let's do the same that for @guillaumemoby, you can use destructuration for all styles juste above the `return`
The same for item.
For the renderItem, you van even have a new function outside the render function

Reply:
I'm not sure at all it is considered a good practice to destructure styles. There are many of them and their names could be misleading once declared in the component namespace.

I can't say i'm an experienced react-native developer but I haven't seen this done anywhere?

Reply:
I went for information on the web and only found this article : https://codereview.stackexchange.com/questions/165936/using-a-destructuring-assignment-on-a-style-object-to-extract-two-members

Keep ieas seem to be :
- "I think it is a matter of preference"
- "You could also compromise and go for s = styles"
- "Also, body or button have unclear semantics without the style. prefix "

I think I would stick to calling `styles.something`

> Clarity and brevity sometimes are at odds. When they are, I choose clarity. 
> -- Jacob Kaplan-Moss

Reply:
Ahah, that's good for me